dispatch/static/manager/src/js/components/ListItemEditorComponentFactory/index.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+import R from 'ramda'
+import DocumentTitle from 'react-document-title'
+
+import ListItemToolbar from './ListItemToolbar'
+
+const NEW_LISTITEM_ID = 'new'
+
+function ListItemEditorComponentFactory(formClass, classString, AFTER_DELETE) {
```

Comment:
As discussed, change this to a component instead of a function. The `formClass`, `classString`, and `AFTER_DELETE` attributes can become members of the component class:

```js
this.formClass
this.classString
this.AFTER_DELETE
```
You should then be able to extend `ListItemEditorComponent` and override the constructor to define those attributes.